docs/source/technical_tutorials/user_data/memory-profiling-python.ipynb

```diff
@@ -0,0 +1,210 @@
+{
```

Comment:
<p>Don't forget to give yourself credit for creating this documentation :) (format similar to other technical guides)</p><p> </p><p>Capitalization in the title and subtitles</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/371/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Agreed! "Author(s):", "Date:", and "Description:" should be listed underneath the title 🙂 maybe to be consistent with other notebooks, change "References:" to a header saying "Additional Resources"</p>